src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
```

Comment:
To correctly display the columns and ensure responsiveness, I would recommend using CSS Grid.
Search for the "CSS Grid Layout Module" to learn more.